app/views/layouts/application.html.erb

```diff
@@ -40,6 +40,19 @@
     <%= content_for?(:body) ? yield(:body) : yield %>
     <%= render 'shared/ajax_modal' %>
     <%= javascript_include_tag 'application' %>
+    <link href="https://cdn.jsdelivr.net/npm/select2@4.0.13/dist/css/select2.min.css" rel="stylesheet" />
+
+    <%= javascript_tag do %>
```

Comment:
This was necessary in order to override some behavior related to https://rubygems.org/gems/almond-rails

Reply:
Upgrading to release 0.3.0 did not seem to properly fix this.